src/test/java/com/amazonaws/services/kinesis/clientlibrary/lib/worker/WorkerTest.java

```diff
@@ -2193,6 +2194,9 @@ private WorkerThreadPoolExecutor getWorkerThreadPoolExecutor() {
         Shard shard1 = ShardObjectHelper.newShard("shardId-1", "shardId-0", null, range1, keyRange);
         shards.add(shard1);
 
+        Shard shard2 = ShardObjectHelper.newShard("shardId-2", null, "shardId-0", range2, keyRange);
```

Comment:
Shouldn't this be `ShardObjectHelper.newShard("shardId-2", "shardId-0", null, range2, keyRange);` since this is the split case and not the merge case? I don't think `shardId-0` is an adjacent shard in this instance.

Reply:
Addressed